docs-src/cloud/tcld/namespace/index.md

```diff
@@ -14,6 +14,7 @@ Alias: `n`
 
 - [tcld namespace create](/cloud/tcld/namespace/create)
 - [tcld namespace delete](/cloud/tcld/namespace/delete)
+- [tcld namespace export s3](/cloud/tcld/namespace/export/index)
```

Comment:
The location is also a bit weird. We should move it after line 22

Reply:
So as a customer read, that could be a bit confusing. tcld namespace create/delete/get/list should be considered as 1 complete set of operations. tcld namespace export is one feature under namespace. 

tcld namespace accepted-client-ca is also not followed a-z order.

Reply:
I would still suggest we change the order of this.